Emby.Server.Implementations/Plugins/PluginManager.cs

```diff
@@ -122,6 +120,12 @@ public IEnumerable<Assembly> LoadAssemblies()
                         ChangePluginState(plugin, PluginStatus.Malfunctioned);
                         continue;
                     }
+                    catch (TypeLoadException ex) // Undocumented exception
+                    {
+                        _logger.LogError(ex, "Failed to load assembly {Path}. This error occurs when a plugin references an incompatible version of one of the shared libraries. Disabling plugin.", file);
+                        ChangePluginState(plugin, PluginStatus.NotSupported);
+                        continue;
+                    }
```

Comment:
```suggestion
                    }
                    catch (Exception ex)
                    {
                        _logger.LogError(ex, "Failed to load assembly {Path}. Unknown exception was thrown. Disabling plugin.", file);
                        ChangePluginState(plugin, PluginStatus.Malfunctioned);
                        continue;
                    }
```

Reply:
Analyzer will fail, need to disable the warning